store-in-gcs/fake-event.py

```
@@ -0,0 +1,296 @@
+
+def generate_event_id() -> str:
+    """Generate a unique event ID"""
+    return str(uuid.uuid4())
+
+
+def get_timestamp() -> str:
+    """Get current timestamp in RFC3339 format"""
+    return datetime.now(timezone.utc).isoformat().replace('+00:00', 'Z')
+
+
+def create_document_fields(fields: Dict[str, Any]) -> Dict[str, Any]:
```

Ruff (0.11.9)

`create_document_fields` is too complex (13 > 10)

(C901)